usecases/memwatch/monitor_test.go

```diff
@@ -149,7 +149,7 @@ func TestMappings(t *testing.T) {
 		}
 	})
 
-	t.Run("check mappings", func(t *testing.T) {
+	t.Run("check mappings, by open many file mappings and close them only after the test is done", func(t *testing.T) {
```

Comment:
**praise (non-blocking):**  thank you!

Reply:
there another `check mappings` test in this file, could you please also add some details about the test behavior

Reply:
added

Reply:
i don't see it. i think it could be you forgot yo push ?